language-server/modules/langserver-core/src/test/resources/completion/statement_context/config/assignment_stmt_ctx_config9.json

```diff
@@ -441,13 +442,13 @@
       "insertTextFormat": "Snippet"
     },
     {
-      "label": "new(string url)",
+      "label": "new(string proxyHost, int proxyPort)",
```

Comment:
This completion item is wrong. It should have been `new(url)`

Reply:
https://github.com/ballerina-platform/ballerina-lang/issues/39413